src/components/Lend/LoanSearch/LoanSearchSavedSearch.vue

```diff
@@ -112,16 +120,26 @@ export default {
 				sector: this.loanSearchState?.sectorId,
 				theme: this.loanSearchState?.themeId.map(themeId => this.themeNames[themeId])
 			};
-		}
+		},
+		loginUrl() {
+			if (this.$route.path === '/') {
+				return '/ui-login';
+			}
```

Comment:
We prod don't need lines 125-127 as this component would never load on the homepage.

Reply:
Fixed, thanks!